aclib/__init__.py

```diff
@@ -34,18 +34,28 @@ def get_dbpath():
 
 
 def get_url(db_name=None, schema_name=None):
-    credential_filename = os.getenv("ACLIB_CREDENTIAL_FILENAME", os.path.expanduser("~") + "/.odbc.ini")
+    username_filename = os.getenv("ACLIB_USERNAME_FILENAME", os.path.expanduser("~") + "/secrets/username")
```

Comment:
The /secrets/... path is an absolute path, not relative to the user home -> please remove os.path.expanduser